src/transformers/models/bert/modeling_bert.py

```diff
@@ -242,24 +246,38 @@ def forward(
         head_mask=None,
         encoder_hidden_states=None,
         encoder_attention_mask=None,
+        past_key_value=None,
         output_attentions=False,
     ):
         mixed_query_layer = self.query(hidden_states)
```

Comment:
(nit) while we're at it I think we could improve the naming here a bit -> I don't think `mixed_query_layer` is good naming `mixed_query_states` would be much better IMO.

Reply:
Agreed, in that case we should also change `key_layer`, `value_layer` to `key_states`, `value_states`. Wdyt ?

Reply:
probably better to do this in separate PR to make the change more explicitly